### PR TITLE
Port few more scripts

### PR DESCRIPTION
**Pull request by blmarket**
## Required sign-off

- [x] I confirm that my PR does not contain any commercial or protected assets and/or source code.
- [x] I agree in advance that my codes will be licensed automatically under the Apache License or similar BSD/MIT-like
      open source licenses in case if OpenNox Project will adopt such a non-GPL license in the future.

### CODE REVIEW

File: src/defs.h [hunk 1]
```diff
@@ -501,6 +501,16 @@ typedef struct {
...
 
+typedef struct {
+	uint32_t field_0;     // 0, 0
+	uint32_t field_1;     // 1, 4
+	uint32_t field_2;     // 2, 8
+	uint8_t field_3;      // 3, 12
+	uint8_t padding_3[3]; // 3, 13
+
+} nox_object_Elevator_data_t;
+// Size unknown
```
Comment (dennwc) [inline src/defs.h]: According to [this](https://github.com/noxworld-dev/opennox/blob/dev/src/object_update.go#L66), size is either 16 or 20, depending on if it's a shaft or the actual elevator.

Else, please consider adding this struct to Go directly (similar to [this](https://github.com/noxworld-dev/opennox/blob/dev/src/server/object_monster.go#L37)).

File: src/script_noxscript_builtins.go [hunk 2]
```diff
@@ -2201,3 +2198,51 @@ func nox_script_ClearOwner_5147E0() int {
 	v1.SetOwner(nil)
 	return 0
 }
+
+func chatTimerFrames(mgr *strman.StringManager, msgId string, obj *Object, durationTicks uint16) {
+		v, _ := strMan.GetVariantInFile(strman.ID(msgId), "CScrFunc.c")
```
Comment (dennwc) [inline src/script_noxscript_builtins.go]:
```suggestion
		v, _ := mgr.GetVariantInFile(strman.ID(msgId), "CScrFunc.c")
```

**Review by dennwc — COMMENTED** [review]
Thank you for the new batch! :rocket: A few minor comments here.

Also, could you please add yourself to `CONTRIBUTORS`?

File: src/script_noxscript_builtins.go (hunk 2 above)
Comment (blmarket) [reply src/script_noxscript_builtins.go]: Added

File: src/defs.h (hunk 1 above)
Comment (blmarket) [reply src/defs.h]: Class is Elevator, so it should be 20.

Re: using Go or C for memory models - Can we defer migration to a later stage?
1. C struct defs can be helpful to use in C code
2. char[] types can be more intuitive with C code

Comment (dennwc) [reply src/defs.h]: Sure, up to you :+1:

**Pull request merged**